opentracing-impl-java8/src/main/java/io/opentracing/AbstractSpan.java

```diff
@@ -127,18 +123,46 @@ public String getBaggageItem(String key) {
     }
 
     @Override
-    public final Span log(String message, /* @Nullable */ Object payload) {
+    public final Span log(String event) {
+        return log(System.nanoTime() / 1000, event);
```

Comment:
yeah, I noticed this the other day, too.

With OT-Java impls I've done (which need to be Java 7 compatible) we've done nanoTime() deltas for duration and millis*1000 for the start timestamp.